protobuf/mayastor.proto

```diff
@@ -500,6 +501,26 @@ message RebuildStatsReply {
   uint64 tasks_active = 7; // number of current active tasks
 }
 
+message RebuildHistoryRequest {
+  string uuid = 1;  // uuid of the nexus
+}
+
+message RebuildHistoryReply {
+  string nexus = 1;                          // Name of the nexus
```

Comment:
Name or Uuid or the nexus?

Reply:
If a nexus is created with a shorthand name it'll be that name, otherwise the autogenerated name like nexus-<uuid>. Basically this information I'm picking from nexus_name() call. Would uuid be preferable here instead of string name?

Reply:
It depends, but on the request you have specifically uuid, and here you have nexus with name in the comment.
Name of the nexus we usually attribute to the volume, and uuid it's usually a uuid specifically for the nexus. I would say always use the uuid rather than the name, or perhaps support both on both the request and reply.

Reply:
Let's keep only uuid in reply as well then.